modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2070,4 +2071,114 @@ describe('V2 Wallet:', function () {
       });
     });
   });
+
+  describe('unspents APIs with string type minValue and maxValue', function () {
+    // using the same test for fanout and consolidate unspent API
+    it('consolidateUnspents should handle string type minValue and maxValue', async function () {
+      const params = { minValue: '123', maxValue: '9000000000000000000123' };
+
+      const scope = nock(bgUrl)
+        .post(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/consolidateUnspents`, body => {
+          return typeof body.minValue === 'string' && body.minValue === '123'
+              && typeof body.maxValue === 'string' && body.maxValue === '9000000000000000000123';
+        })
+        .reply(200, { txInfo: { unspents: [{ id: 123, address: 'sfajlkjad', value: 1242123, valueString: '1242123' }] } });
+
+      wallet.keyIds().forEach((keyId) => nock(bgUrl)
+        .get(`/api/v2/${wallet.coin()}/key/${keyId}`)
+        .reply(200, {}));
+
+      const stub = sinon.stub(wallet, 'signTransaction').resolves({});
+
+      const sendScope = nock(bgUrl)
+        .post(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/tx/send`, { type: 'consolidate' })
+        .reply(200, {});
+
+      await wallet.consolidateUnspents(params);
+
+      const txInfo = stub.getCall(0)?.args?.[0]?.txPrebuild?.txInfo as any;
+      txInfo?.unspents?.[0].value.should.be.a.type('bigint');
+
+      scope.isDone().should.be.True();
+      sendScope.isDone().should.be.True();
+    });
+
+    it('maxspendable should handle string type minValue and maxValue', async function () {
+      const params = { minValue: '123', maxValue: '9000000000000000000123', target: '9000000000000000000123' };
+
+      const scope = nock(bgUrl)
+        .get(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/maximumSpendable`)
+        .query(params => {
+          sinon.assert.match(params.minValue, '123');
+          sinon.assert.match(params.maxValue, '9000000000000000000123');
+          sinon.assert.match(params.target, '9000000000000000000123');
+          return params;
+        })
+        .reply(200, {});
+
+      await wallet.maximumSpendable(params);
+
+      scope.isDone().should.be.True();
+    });
+
+    it('get unspents should handle string type minValue and maxValue', async function () {
+      const params = { minValue: '123', maxValue: '9000000000000000000123', target: '9000000000000000000123' };
+
+      const scope = nock(bgUrl)
+        .get(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/unspents`)
+        .query(params => {
+          sinon.assert.match(params.minValue, '123');
+          sinon.assert.match(params.maxValue, '9000000000000000000123');
+          sinon.assert.match(params.target, '9000000000000000000123');
+          return params;
+        })
+        .reply(200, {});
+
+      await wallet.unspents(params);
+
+      scope.isDone().should.be.True();
+    });
+    nock.cleanAll();
```

Comment:
This statement does not get executed after `it` is run. It needs to be in an `after()` block